pkg/backend/snapshot.go

```diff
@@ -653,7 +640,7 @@ func (sm *SnapshotManager) snap() *deploy.Snapshot {
 
 	// if any resources have been deleted we need to filter them out here
```

Comment:
This comment needs updating. Maybe something like:

```
// If any resources are "done", we need to filter them out here.  This applies to any "Old" resources, including deleted ones.  They should not end up in the snapshot
```

Is that understanding correct?

Reply:
Will update